src/intrprtr.c

```diff
@@ -82,8 +83,32 @@
 */
 /* TL: UInt IntrCoding; */
 
+// INTERPRETER_PROFILE_HOOK deals with profiling of immediately executed
+// code.
+// If STATE(IntrCoding) is true, profiling is handled by the bytecode
+// generation and interpreter, so skip. Otherwise, we mark the line as
```

Comment:
It is misleading to talk about "bytecode" here, GAP doesn't generate bytecode but rather an (A)ST.

Reply:
changed